### ISSUE TITLE
Rails 5 compatibility

### ISSUE THREAD

**Issue by coderhs** [marketplacer/lightspeed_pos#19]
HI there I am building an intergration on Rails 5. But I am having a slight issue where in when i do a bundle install its giving me the following errors. Is it because active_support to set to `4.2` any issues if we change it to `5.+`

```
Bundler could not find compatible versions for gem "activesupport":
  In snapshot (Gemfile.lock):
    activesupport (= 5.0.2)

  In Gemfile:
    carrierwave (~> 1.0) was resolved to 1.0.0, which depends on
      activesupport (>= 4.0.0)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activejob (= 5.0.2) was resolved to 5.0.2, which depends on
        globalid (>= 0.3.6) was resolved to 0.3.7, which depends on
          activesupport (>= 4.1.0)

    jbuilder (~> 2.5) was resolved to 2.6.3, which depends on
      activesupport (< 5.2, >= 3.0.0)

    lightspeed_pos was resolved to 0.3.0, which depends on
      activesupport (~> 4.2.0)

    lograge was resolved to 0.4.1, which depends on
      activesupport (< 5.1, >= 4)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      activesupport (= 5.0.2)

    jquery-rails was resolved to 4.2.2, which depends on
      rails-dom-testing (< 3, >= 1) was resolved to 2.0.2, which depends on
        activesupport (< 6.0, >= 4.2.0)

    rails_best_practices was resolved to 1.18.0, which depends on
      activesupport

    spring was resolved to 2.0.1, which depends on
      activesupport (>= 4.2)

    rails (>= 5.0.0.1, ~> 5.0) was resolved to 5.0.2, which depends on
      sprockets-rails (>= 2.0.0) was resolved to 3.2.0, which depends on
        activesupport (>= 4.0)
```

I am referencing to this github repo and not the gem, in my project.

**Comment by mipearson** [comment]
Hi,

I'd gladly accept a pull request that fixes this, if you can test that this continues to work with AS 5.x

**Comment by mipearson** [comment]
Fixed by #20
